.github/workflows/sycl_nightly.yml

```diff
@@ -27,6 +27,12 @@ jobs:
       lts_matrix: ${{ needs.test_matrix.outputs.lts_lx_matrix }}
       lts_aws_matrix: ${{ needs.test_matrix.outputs.lts_aws_matrix }}
 
+      # We use the build for the release, override its name and prefer
+      # widespread gzip compression.
```

Comment:
For instance, LLVM uses LZMA - https://github.com/llvm/llvm-project/releases/tag/llvmorg-16.0.0.

Is `gzip` "widespread" on Windows too?